app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ActiveRecord::Base
+  validates_presence_of :lat
```

Comment:
Forgot about this, but we do associations before validations: https://github.com/IntrepidPursuits/web-style-guides/blob/master/style.md#rails